Cargo.toml

```diff
@@ -12,6 +12,7 @@ members = [
   "link-crypto",
   "link-git-protocol",
   "link-identities",
+  "linkd-lib",
```

Comment:
`link-daemon`?

Reply:
I liked the `blink`

Reply:
In my understanding `linkd-lib` is adhering to our non-RFCed standard of crate naming, `linkd` being the prefix of the binary that consumes this crate (same as the `rad` crates for the cli).

For comedic purpose and artificial increase of barrier to entry for contributors I'm not opposed to naming it `dlink`.

Reply:
> the prefix of the binary that consumes this crate

Interesting, this interpretation didn't occur to me. I'm thinking of `rad` as an anagram of `cli`, where `cli` is a bunch of commandline executables, not necessarily that they must be invoked by typing `rad` first (the fact that we said it will be one executable for starters is an implementation detail if you will). I'm also thinking of these prefixes as a navigational aid in a monorepo structure, while I discarded the idea to enforce complete separation -- I would expect that there will always be things which seem worthwhile to put in their own crate, so they can be depended on in multiple places.

So tl;dr I feel `linkd-` as a top-level name is too specific, because we may have other daemons. Maybe we can find a different prefix conveying daemon-ness? It could be `daemon-`, although speling...

Reply:
> different prefix conveying daemon-ness?

How about `matt`? 

![image](https://user-images.githubusercontent.com/3503528/132823473-11d3d829-a1c4-4468-a60a-9230e4e97a0e.png)

Reply:
3d4e38c